zerver/lib/send_email.py

```diff
@@ -91,10 +91,11 @@ class EmailNotDeliveredException(Exception):
 
 # When changing the arguments to this function, you may need to write a
 # migration to change or remove any emails in ScheduledEmail.
-def send_email(template_prefix: str, to_user_id: Optional[int]=None, to_email: Optional[str]=None,
-               from_name: Optional[str]=None, from_address: Optional[str]=None,
-               reply_to_email: Optional[str]=None, context: Dict[str, Any]={}) -> None:
-    mail = build_email(template_prefix, to_user_id=to_user_id, to_email=to_email, from_name=from_name,
+def send_email(template_prefix: str, to_user_ids: Optional[List[int]]=None,
```

Comment:
This API change requires some corresponding changes in the `deliver_email.py` management command, which I went ahead and made.